src/main/java/seedu/command/AddIncomeCommand.java

```diff
@@ -11,24 +14,37 @@ public class AddIncomeCommand extends AddTransactionCommand {
     public static final String[] COMMAND_MANDATORY_KEYWORDS = {"a/"};
     public static final String[] COMMAND_EXTRA_KEYWORDS = {"d/"};
 
-    public AddIncomeCommand(TransactionList transactions, String description, String amount, String date) {
-        super(transactions, description, amount, date);
+    public AddIncomeCommand(TransactionList transactions) {
+        super(transactions);
     }
 
     @Override
     public List<String> execute() throws Exception{
```

Comment:
Instead of throwing exception, you may handle this exception in the execution (as this is a man-made exception for parsing wrong date time string). Check ViewIncomeCommand to see how the exception is handled